src/sass/_uswds-theme-custom-styles.scss

```diff
@@ -97,3 +97,158 @@ i.e.
     max-width: 50%;
   }
 }
+
+// Buttons
+.usa-button {
+  height:units(6);
```

Comment:
[Pebble] SUPER lame comment, but could you align the spacing across this file? Seems like there's a few instances where there's a missing space between the css property and value. (Meta-note: I would expect the IDE to auto-format, is it working properly?)